Subject: [PATCH] Patch swagger initializer to fetch openAPI from document URL
---
Index: dist/swagger-initializer.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/dist/swagger-initializer.js b/dist/swagger-initializer.js
--- a/dist/swagger-initializer.js	(revision 872fc3a76b71063f57f99fac8bd50da135cbe7b8)
+++ b/dist/swagger-initializer.js	(date 1730793908758)
@@ -2,8 +2,11 @@
   //<editor-fold desc="Changeable Configuration Block">

   // the following lines will be replaced by docker/configurator, when it runs in a docker-container
+
+  var openApiUrl = document.URL.substr(0,document.URL.indexOf("swagger-ui/")) + "openapi.yml";
+
   window.ui = SwaggerUIBundle({
-    url: "https://petstore.swagger.io/v2/swagger.json",
+    url: openApiUrl,
     dom_id: '#swagger-ui',
     deepLinking: true,
     presets: [
